.github/workflows/basic-eval.yml

```diff
@@ -0,0 +1,20 @@
+name: Basic evaluation checks
+
+on:
+  pull_request:
+    branches:
+     - master
+     - release-**
+  push:
+    branches:
+     - master
+     - release-**
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    # we don't limit this action to only NixOS repo since the checks are cheap and useful developer feedback
+    steps:
+    - uses: actions/checkout@v2
+    - uses: cachix/install-nix-action@v13
+    # explicit list of supportedSystems is needed until aarch64-darwin becomes part of the trunk jobset
+    - run: nix-build pkgs/top-level/release.nix -A tarball.nixpkgs-basic-release-checks --arg supportedSystems '[ "aarch64-darwin" "aarch64-linux" "x86_64-linux" "x86_64-darwin"  ]'
```

Comment:
Would gives us a build matrix a 4x speedup here?

Reply:
We also need to make sure people to not try to run this command on an older nixpkgs checkout or they are greeted with this error message: ``error: attribute 'nixpkgs-basic-release-checks' in selection path 'tarball.nixpkgs-basic-release-checks' not found``

Reply:
Could that happen?  This `.yml` is in-repo, so I don't expect it would naturally act on older commits.

Reply:
🤔 I think you are right. People wouldn't run into this when they use an older checkout.

Reply:
> Would gives us a build matrix a 4x speedup here?

Not entirely since there's overhead of the job doing other checks as well.

I'm a bit hestiant to do that not to hit the jobs limit on our repo.

Reply:
What is the job limit?

Reply:
[20 concurrent](https://docs.github.com/en/actions/reference/usage-limits-billing-and-administration)